### ISSUE TITLE
'Analytics/SEGAnalytics.h' file not found

### ISSUE THREAD

**Issue by jeremynorman89** [claimsforce-gmbh/flutter-segment#99]
After the upgrade to `3.4.1` I am now getting the following error:

`pub-cache/hosted/pub.dartlang.org/flutter_segment-3.4.1/ios/Classes/FlutterSegmentPlugin.m:2:9: fatal error: 'Analytics/SEGAnalytics.h' file not found`

I've tried the usual troubleshooting steps such as `flutter clean`, deleting derived data etc. Any idea why this would be happening?

**Comment by nmarko91** [comment]
Same, I tried removing `podfile lock` & running `flutter clean` after but I have the same issue :(

**Comment by KishanAcko** [comment]
Am also facing the same issue

**Comment by juliavi** [comment]
Same problem here. tried -
flutter clean
pod deintegrate
reinstalling / upgrading / restarting....

**Comment by mahmutpinarbasi** [comment]
after upgrading 3.4.1, started to have same error.

**Comment by yoodame** [comment]
Hello all. I was able to bypass this issue partly by replacing the Analytics imports in `ios/Classes/FlutterSegmentPlugin.m` with:

```
#import <Segment/SEGAnalytics.h>
#import <Segment/SEGContext.h>
#import <Segment/SEGMiddleware.h>
```

I am still trying to figure out why the Segment-Amplitude import is failing.

**Comment by esarbanis** [comment]
Just submitted a PR for this.

**Comment by jeremynorman89** [comment]
> Just submitted a PR for this.
> 
> You can switch to the PR's branch **_temporarily_** if you do not want to be blocked until it is merged.
> 
> ```
> flutter_segment: 
>     git:
>       url: git@github.com:esarbanis/flutter-segment.git
>       ref: b56dc4fea1428cdbc5745137d207b420cbbe57d5
> ```

Everything seems to be working using this branch, thanks! 👍  Hopefully it can get merged soon.

**Comment by juliavi** [comment]
Any estimation when this would be merged? :)

**Comment by esarbanis** [comment]
@juliavi if you do not have any ssh key setup with github, then you should use: `https://github.com/esarbanis/flutter-segment.git` url

**Comment by MaiKaY** [comment]
Fixed in `3.5.0` and please keep the following in mind.

> Since version `3.5.0` we are forcing all users to use `use_frameworks!` within the [`Podfile`](https://github.com/claimsforce-gmbh/flutter-segment/blob/master/example/ios/Podfile#L31) due to import issues of some 3rd party dependencies.
